infra/resources/modules/commons/api/ts_management/v1/_openapi.yaml

```diff
@@ -273,6 +304,35 @@ components:
       schema:
         type: string
         minLength: 1
+    queryPageSize:
+      name: pageSize
+      in: query
+      required: false
+      schema:
+        type: integer
+        minimum: 1
+        maximum: 100
+        default: 25
+    queryMaximumId:
+      name: maximumId
+      in: query
+      required: false
+      schema:
+        type: string
+        maxLength: 26
+        minLength: 26
+        pattern: '[0-9a-hjkmnp-tv-zA-HJKMNP-TV-Z]{26}'
```

Comment:
**suggestion**: i don't think this is needed

Reply:
Removed regex validation in 71323943e0b5b6de4d732d5465c4bafa2973e4b1